pages/phases/job-analysis/defining-proficiency-levels.md

```diff
@@ -6,48 +6,91 @@ title: Defining Proficiency Levels and Required Experience
 
-- **Scale of activity:** The applicant did something at a small organization _-vs-_ did something at a large organization.
-- **Seniority:** The applicant did something as part of a group _-vs-_ lead the group that did something.
+- **Complexity:** The applicant can manage straightforward workflows _-vs-_ can manage highly complex workflows.
 - **Level of detail:** The applicant can explain a general concept _-vs-_ can discuss the concept in detail.
+- **Scale of activity:** The applicant did something at a small organization _-vs-_ did something at a large organization.
+- **Seniority/independence:** The applicant did something as part of a group _-vs-_ lead the group that did something.
 
-### Setting the required proficiency level
+When defining proficiency levels, you have two options.
+
+### Two levels
+
+Use two levels if your announcement is only for one GS grade and you want to save time during job analysis by defining the minimum required levels: the minimum required level and one level above that to establish categories as required by category rating rules. If the hiring action was for a GS13 position, the “meets” level would mean the applicant needs 52 weeks experience at that level in order to qualify for GS13. If your hiring action is for two GS grades, you will need at least three proficiency levels.
+
+#### Example (for an analytical competancy)
+
+* **GS-13 Meets:** Breaks problems down into component parts. Displays repeated
+experience in qualitative and quantitative analysis. Defines product or project metrics
+beyond the basics and ties these metrics to decisions.
+* **GS-13 Exceeds:** Brings analytical thinking to everything they do, and has a range and depth of
+experience doing so. Anticipates the need for metrics and analysis early in the product
+and project discovery and design process, and carries metrics through iterations.
+
+### Four levels
+
+Use four levels when you may want to leverage this work for other grades in the future; the same set of competencies and proficiencies can be used for different grade levels because roles that require more seniority set the required proficiencies higher than junior roles. It may also help to define four levels before deciding which level is appropriate for the grade in question. If you use four levels, we recommend these titles for each level:
+
+1. Novice
+2. Competant
+3. Advanced
+4. Expert
+
+Once proficiency levels are defined, HR and the SMEs will decide at which level you would require applicants to have one year of experience in order to qualify for the grade in question. For example, they made decide that to qualify for a GS13, applicants must have one year experience at the “Advanced” level. See this example:
+
+#### Example (for an analytical competancy)
+
+* **Novice:** Unable to break down problems or only talks through problems at a high
```

Comment:
@wslack -- How could this level, as defined, be helpful in a future action? Would we want to hire a GS-11 who actively eschews data in decision-making?

Reply:
the four competency model makes you write  down what someone unqualified looks like, which in theory helps to sharpen the others. 

We can switch it back to "collaboration" but I'm also good using something else from the library (this is what's in the [library's explanation](https://smeqa.usds.gov/toolkit/job-analysis/sample-competencies-proficiencies.pdf))

Reply:
Right, but I think it's unqualified for a GS-13... not unqualified for government work in general. I think actively eschewing data in decision making is like a critical fail. So yeah, I think "collaboration" is a good change.

Reply:
k